web/profiles/custom/os2loop/modules/os2loop_oembed/src/TwigExtension/TwigExtension.php

```diff
@@ -180,7 +180,7 @@ private function createVideoFromUrl($text): array {
         // If oembed is not an option create iframe from a url.
         elseif ('custom' === self::ALLOWED_PROVIDERS[$url['host']]['type']) {
           $video['custom']['src'] = $text;
-          $video['iframe'] = '<iframe src="' . $text . '"></iframe>';
+          $video['iframe'] = '<iframe allowfullscreen src="' . $text . '"></iframe>';
```

Comment:
`allowfullscreen` “is considered a legacy attribute” (cf. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen).
Use https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy/Using_Feature_Policy#the_iframe_allow_attribute.

Reply:
Changed